docs/120-homebrew.md

```diff
@@ -78,6 +83,12 @@ brew:
     system "#{bin}/program --version"
     ...
 
+  # Specify any additional formula content in the special section.
+  # Default is empty.
+  special: |
```

Comment:
this is not a homebrew thing, right?

the idea is to use this to add arbitrary instructions to the formula?

if yes, I think this is the indication of other missing features...

Reply:
Yes, that's correct. We used to use this in the past but actually don't use it anymore. So I think for our use case we don't really need it.